package.json

```diff
@@ -106,9 +106,6 @@
     "transformIgnorePatterns": [
       "node_modules/(?!(jest-)?react-native|react-navigation)"
     ],
-    "moduleNameMapper": {
-      "\\.(png|gif|jpeg|jpg)$": "RelativeImageStub"
-    },
```

Comment:
Unfortunately this piece seems to be needed to run the tests on Windows. Otherwise you get errors like this when an image is loaded in a test:

```
 FAIL  src/screens/FilterScreen/Header.test.js
  ● Test suite failed to run

    C:/code/pride-london-app/assets/images/chevron-left-white.png: Unexpected character '�' (1:0)

      > 1 | �PNG
          | ^
        2 | →
        3 |
        4 | IHDR   0   ♠   W☻��   ☺sRGB ��∟�   �IDATh♣�I
```